.gitignore

```diff
@@ -125,6 +125,7 @@ numpy/core/include/numpy/multiarray_api.txt
 numpy/core/include/numpy/ufunc_api.txt
 numpy/core/lib/
 numpy/core/src/common/npy_binsearch.h
+numpy/core/src/common/npy_cpu_features.c
```

Comment:
Doesn't really belong in this PR.

Reply:
Should I break it out into a separate commit?

Reply:
Yeah, that sounds good - can keep it in the same PR for simplicity.

Reply:
why is this needed? That file (and some of the others here) should be generated in the `build/` directory, not in the main directory

Reply:
When you do `python setup.py develop` for local testing it creates that file in this directory.